apps/strapi-cms/config/plugins.ts

```diff
@@ -3,14 +3,17 @@ export default ({ env }: any) => ({
     config: {
       provider: env('UPLOAD_PLUGIN_PROVIDER', 'aws-s3'),
       providerOptions: {
-        accessKeyId: env('AWS_ACCESS_KEY_ID'),
-        secretAccessKey: env('AWS_ACCESS_SECRET'),
-        region: env('AWS_REGION'),
-        params: {
-          ACL: 'private',
-          signedUrlExpires: 15 * 60,
-          Bucket: env('AWS_BUCKET_NAME'),
-        },
+        baseUrl: env('CDN_URL'),

```

Comment:
Does this configuration (`baseUrl`) work with both local providers and the S3 provider?

Reply:
I don't believe it does. I haven't tested it though.

It's mentioned in the [docs for the AWS provider](https://market.strapi.io/providers/@strapi-provider-upload-aws-s3) so I would assume it's meant for that.

The docs for these providers are poor at best honestly.